sources/imperative/reader/ReaderAPI.swift

```diff
@@ -220,6 +271,11 @@ fileprivate extension CSVReader.Error {
               help: "Make sure the URL is valid and you are allowed to access the file. Alternatively set the configuration's presample or load the file in a data blob and use the reader's data initializer.",
               userInfo: ["File URL": url])
     }
+    /// Error raised when an input stream cannot be created for "/dev/stdin".
+    static func _invalidStdin() -> CSVError<CSVReader> {
+        .init(.streamFailure,
+              reason: "Creating an input stream from stdin failed.")
```

Comment:
Try to add `help` cues. Even if they are a bit stupid, such as "Please retry again".